grub-core/script/execute.c

```diff
@@ -943,6 +945,25 @@ grub_script_execute_cmdline (struct grub_script_cmd *cmd)
   if (grub_script_arglist_to_argv (cmdline->arglist, &argv) || ! argv.args[0])
     return grub_errno;
 
+  for (i = 0; i < argv.argc; i++) {
+	  cmdlen += grub_strlen (argv.args[i]) + 1;
+  }
+
+  cmdstring = grub_malloc (cmdlen);
+  if (!cmdstring)
+  {
+	  return grub_error (GRUB_ERR_OUT_OF_MEMORY,
+			     N_("cannot allocate command buffer"));
+  }
+
+  for (i = 0; i < argv.argc; i++) {
+	  offset += grub_snprintf(cmdstring + offset, cmdlen - offset, "%s ",
```

Comment:
I don't know what's going on in grub, either always have a space after the function name or don't... it looks like the rest of this file has a space, so this and the call a few lines below should have spaces I guess.

Some of the listings seemed predominantly space-less, maybe those need to be changed to have spaces, *shrug*